File: Java/Processor.java
public class Processor {
    private String name;    // menampung nama processor
    private int price;      // menampung harga

    // constructor
    public Processor() {
    }

    public Processor(String name, int price) {
        this.name = name;
        this.price = price;
    }

    // setter and getter
    public void setName(String name) {
        this.name = name;
    } 

    public String getName() {
        return this.name;
    }

    public void setPrice(int price) {
        this.price = price;
    }

    public int getPrice() {
        return this.price;
    }

    // method
    public void printProcessor() {
        System.out.println("Nama Processor : " + this.getName());
        System.out.println("Harga Processor : Rp " + this.getPrice());
    }
}

File: Python/Processor.py
class Processor:
    #variabel untuk menampung nama prosesor dan harganya
    __name = ""
    __price = 0

    # constructor
    def __init__(self, name = "", price = 0):
        self.__name = name
        self.__price = price

    # setter and getter
    def setName(self, name):
        self.__name = name
    
    def getName(self):
        return self.__name

    def setPrice(self, price):
        self.__price = price

    def getPrice(self):
        return self.__price

    # method
    def printProcessor(self):
        print("Nama Processor : " + str(self.getName()))
        print("Harga Processor : Rp " + str(self.getPrice()))